s3backgrounddelete/kafka_setup.md

```diff
@@ -0,0 +1,34 @@
+### License
+
```

Comment:
All the kafka related code should be checked into cortx-utils and it should be independent of S3. There should be clearly carved out interfaces to CORTX Message bus where S3 background delete can push or subscribe for the messages. Interfaces needs to be generic and independent of S3.